Source/OxyPlot/Series/BarSeries/RectangleBarSeries.cs

```diff
@@ -111,10 +116,10 @@ public override TrackerHitResult GetNearestPoint(ScreenPoint point, bool interpo
                 var r = this.ActualBarRectangles[i];
                 if (r.Contains(point))
                 {
-                    double value = (this.Items[i].Y0 + this.Items[i].Y1) / 2;
+                    double value = (this.ActualItems[i].Y0 + this.ActualItems[i].Y1) / 2;
                     var sp = point;
                     var dp = new DataPoint(i, value);
-                    var item = this.GetItem(i);
+                    var item = this.ActualItems[i];
```

Comment:
(I think this could be moved to L119 and all references to `ActualItems` be replaced by `items`)